src/osx/cocoa/menuitem.mm

```diff
@@ -270,6 +270,15 @@ void Hide( bool hide ) wxOVERRIDE
             wxLogDebug("wxMenuItemCocoaImpl::Hide not yet supported under OS X < 10.5");
     }
 
+    void SetAllowsKeyEquivalentWhenHidden( bool allow ) wxOVERRIDE
+    {
+        // setAllowsKeyEquivalentWhenHidden is available since macOS 10.13
+        if (@available(macOS 10.13, *))
```

Comment:
We use `WX_IS_MACOS_AVAILABLE()` from `wx/osx/private/available.h` for such checks. I'm not sure if the rationale there is still valid, but I think it might be, and it would be better to use the macro here too if only for consistency.